python/packages/autogen-core/src/autogen_core/components/models/_openai_client.py

```diff
@@ -526,7 +552,9 @@ async def create_stream(
             )
         else:
             stream_future = asyncio.ensure_future(
-                self._client.chat.completions.create(messages=oai_messages, stream=True, **create_args)
```

Comment:
Does the beta client also support stream mode?

Reply:
streaming option is available but not as an argument and not sure if this is going to be standard way moving forward
reference: https://github.com/openai/openai-python/blob/main/src/openai/resources/beta/chat/completions.py#L236

should i remove the beta client when streaming is enabled?